docs/overview/office-add-in-availability.md

```diff
@@ -340,19 +340,19 @@ To work as expected, your Office Add-in might depend on a specific Office applic
   <tr>
     <td>Office on the web</td>
     <td>- Custom Functions</td>
-    <td>- <a href="/office/dev/add-ins/excel/custom-functions-requirement-sets">CustomFunctionsRuntime 1.1</a></td>
+    <td>- <a href="/office/dev/add-ins/excel/custom-functions-requirement-sets">CustomFunctionsRuntime 1.3</a></td>
```

Comment:
Other nodes on this page list out every requirement set (like ExcelApi 1.1, 1.2, ...). I think this should match and list 1.1, 1.2, and 1.3 for the different platforms.

Reply:
I see your point. I will update this PR. Thanks for the feedback, @AlexJerabek!